src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -273,12 +273,7 @@ public static Pageable createPageableObject(Integer page, Integer size, Sort sor
     }
 
     public static String getShowDeletedValue(String showDeleted) {
-        if ("True".equalsIgnoreCase(showDeleted)) {
-            showDeleted = "true";
-        } else {
-            showDeleted = "false";
-        }
-        return showDeleted;
+        return "True".equalsIgnoreCase(showDeleted) ? ProfessionalApiConstants.TRUE : ProfessionalApiConstants.FALSE;

```

Comment:
may be we can use Boolean.True as boolean constant. if we want to compare boolean values.

Reply:
sure . I will do that